news/_posts/core-weekly/2022-08-01-coreweekly-30-2022.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 30 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-08-01
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 25th to Sunday 31th of July 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+With the help of the community, the maintainers team focused on the PrestaShop 8.0.0 Beta release. You can follow the release process [here](https://github.com/PrestaShop/PrestaShop/issues/26427). The work continues, but we expect a Beta version very soon. Stay tuned!
+
+A few weeks ago, we started organizing cyclic meetings of maintainers and committers, where we discuss various topics to help the project move forward and address some issues. One we discussed was the process of releasing a new version of modules.
+The effects of the discussion are already visible! @matks [created an application](https://github.com/PrestaShop/ps-monitor-module-releases) that generates a page available for everyone where we can monitor which modules should be released. This will help us better monitor this process.
```

Comment:
```suggestion
The effects of the discussion are already visible! @matks [created an application](https://github.com/PrestaShop/ps-monitor-module-releases) that generates [a page](https://build.prestashop.com/ps-monitor-module-releases/) available for everyone where we can monitor which modules should be released. This will help us better monitor this process.
```